lib/index.js

```diff
@@ -1,20 +1,20 @@
-// index module
+'use strict'
 
-var constants = require('./constants')
-var Server = require('./server')
-var runner = require('./runner')
-var stopper = require('./stopper')
-var launcher = require('./launcher')
-var cfg = require('./config')
+const constants = require('./constants')
+const Server = require('./server')
+const runner = require('./runner')
+const stopper = require('./stopper')
+const launcher = require('./launcher')
+const cfg = require('./config')
 
 // TODO: remove in 1.0
```

Comment:
It's 2.0 already :)
Shall we remove this method?

Reply:
yes again in a separate PR.